libsolidity/parsing/Parser.cpp

```diff
@@ -1240,13 +1240,13 @@ ASTPointer<Mapping> Parser::parseMapping()
 	}
 	else
 		fatalParserError(1005_error, "Expected elementary type name or identifier for mapping key type");
-	ASTPointer<ASTString> keyName = make_shared<ASTString>("");
+	ASTPointer<ASTString> keyName = std::make_shared<ASTString>("");
 	SourceLocation keyNameLocation{};
 	if (m_scanner->currentToken() == Token::Identifier)
 		tie(keyName, keyNameLocation) = expectIdentifierWithLocation();
 	expectToken(Token::DoubleArrow);
 	ASTPointer<TypeName> valueType = parseTypeName();
-	ASTPointer<ASTString> valueName = make_shared<ASTString>("");
+	ASTPointer<ASTString> valueName = std::make_shared<ASTString>("");
 	SourceLocation valueNameLocation{};
 	if (m_scanner->currentToken() == Token::Identifier)
 		tie(valueName, valueNameLocation) = expectIdentifierWithLocation();
```

Comment:
I'm rebasing my PRs on top of this now and just noticed that there are quite a few instances of `std::tie()` without `std::` here. How did that get through `check_style.sh`?